cli.js

```diff
@@ -31,17 +32,30 @@ if (args.help || args._.length === 0) {
   console.log("--ignore-module, -i   Won't tell you about module names passed in as --ignore-module / -i. Only usable with --unused")
   console.log('--entry               By default your main and bin entries from package.json will be parsed, but you can add more the list of entries by passing them in as --entry')
   console.log("--no-default-entries  Won't parse your main and bin entries from package.json will be parsed")
+  console.log('--detective           Pointing to a requireable path of a javascript file containing an alternative implementation of the detective module that supports extended or alternate syntaxes')
+  console.log("--extensions, -e      Comma-separated list of file extensions to use when resolving require paths. Eg. 'js,jsx'")
   console.log('--version             Show current version')
   console.log('--ignore              To always exit with code 0 pass --ignore')
   console.log('')
 
   process.exit(1)
 }
 
+var extensions
+
+if (args.e) {
+  extensions = args.e.split(',').map(function (item) {
+    item = item.trim()
+    return item[0] === '.' ? item : '.' + item
+  })
+}
+
 check({
   path: args._.shift(),
   entries: args._.concat(args.entry || []),
-  noDefaultEntries: !args['default-entries']
+  noDefaultEntries: !args['default-entries'],
+  extensions: extensions,
```

Comment:
Can we call a function here instead? E.g. `split(args.extensions)`? May also want to handle it as an array since you can do `-e .js -e .jsx`.